zebra-network/src/peer_set/candidate_set.rs

```diff
@@ -106,7 +106,7 @@ where
         // existing peers, but we don't make too many because update may be
         // called while the peer set is already loaded.
         let mut responses = FuturesUnordered::new();
-        for _ in 0..2usize {
+        for _ in 0..1usize {
```

Comment:
Is this just length 1?

Reply:
Yes, we could also cut out the loop entirely and just make one request, at the cost of it being more work to add fanout back in later.

Reply:
```suggestion
        // Yes this loops only once (for now), until we add fanout back.
        for _ in 0..1usize {
```

Reply:
Happy to do this or just cut out the loop, whichever is preferable.

Reply:
I'm fine with keeping the loop if we add the comment 👍